docs/admin/reference/hardware.rst

```diff
@@ -125,5 +125,15 @@ Qubes compatible, the HCL is a good choice.
 
 Choosing a printer
 ------------------
-In order to print submissions, a supported non-networked printer is required. We have tested and recommend the HP LaserJet Pro M404n. More printer options will be added in future releases.
+SecureDrop Workstation only supports printing over USB, and so it is essential to choose a compatible printer that **has a USB port**. 
 
+Driverless IPP printing is used to support a large selection of modern printers. Compatible printers can be easily identified by their support for the Apple AirPrint or Moipra standards:
+
+.. figure:: images/airprint.jpg
+
+.. figure:: images/moipra.jpg
+
+You may consult Apple's `list of printers that support AirPrint <https://support.apple.com/en-us/HT201311#printers>`_, Moipra's `list of certified products <https://mopria.org/certified-products>`_, or OpenPrinting's `list of printers supporting driverless printing <https://openprinting.github.io/printers/>`_, looking for a compatible printer that will print over USB. 
```

Comment:
```suggestion
To find a printer that supports driverless IPP printer over USB, you may consult Apple's `list of printers that support AirPrint <https://support.apple.com/en-us/HT201311#printers>`_, Moipra's `list of certified products <https://mopria.org/certified-products>`_, or OpenPrinting's `list of printers supporting driverless printing <https://openprinting.github.io/printers/>`_.
```
Recommended rearranging this sentence for clarity

Reply:
Ah, that's an interesting distinction! In that case I think you're right that my suggestion is not an improvement.

There is something about the original wording that my brain has a hard time wrapping around. Perhaps if we changed it to something like "..., being careful to choose a model that provides a USB port" rather than "looking for a compatible printer that will print over USB."

This is obviously a loose suggestion, and I defer to you as well as the other reviewers.